test/smoke/run-smoke-suite.sh

```diff
@@ -0,0 +1,239 @@
+#!/bin/bash
+
+if [[ -n "${DEBUG}" ]]; then
+  set -x
+fi
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+export KIND_CLUSTER_NAME=${KIND_CLUSTER_NAME:-ingress-smoke-test}
+export KUBECONFIG="${KUBECONFIG:-$HOME/.kube/kind-config-$KIND_CLUSTER_NAME}"
+
+# Variables required for pulling Docker image with pytest
+SMOKE_REGISTRY_NAME="${SMOKE_REGISTRY_NAME:-dkr.wallarm.com}"
+SMOKE_IMAGE_PULL_SECRET_NAME="pytest-registry-creds"
+
+SMOKE_IMAGE_NAME="${SMOKE_IMAGE_NAME:-dkr.wallarm.com/tests/smoke-tests}"
+SMOKE_IMAGE_TAG="${SMOKE_IMAGE_TAG:-latest}"
+
+# Allure related variables
+ALLURE_ENDPOINT="${ALLURE_ENDPOINT:-https://allure.wallarm.com}"
+ALLURE_PROJECT_ID=${ALLURE_PROJECT_ID:-10}
+ALLURE_RESULTS="${ALLURE_RESULTS:-/tests/_out/allure_report}"
+ALLURE_UPLOAD_REPORT="${ALLURE_UPLOAD_REPORT:-false}"
+ALLURE_GENERATE_REPORT="${ALLURE_GENERATE_REPORT:-false}"
+
+# Pytest related variables
+CLIENT_ID="${CLIENT_ID:-5}"
+WALLARM_API_CA_VERIFY="${WALLARM_API_CA_VERIFY:-true}"
+WALLARM_API_HOST="${WALLARM_API_HOST:-api.wallarm.com}"
+WALLARM_API_PRESET="${WALLARM_API_PRESET:-eu1}"
+NODE_BASE_URL="${NODE_BASE_URL:-http://workload.default.svc.cluster.local}"
+PYTEST_ARGS=$(echo "${PYTEST_ARGS:---allure-features=Node}" | xargs)
+PYTEST_WORKERS="${PYTEST_WORKERS:-10}"
+#TODO We need it here just to don't let test fail. Remove this variable when test will be fixed.
+HOSTNAME_OLD_NODE="smoke-tests-old-node"
+
+NODE_VERSION=$(< "${CURDIR}/AIO_BASE" awk -F'[-.]' '{print $1"."$2"."$3}')
```

Comment:
We need to get node version from here
https://github.com/wallarm/sidecar/blob/main/helm/values.yaml#L186

Reply:
done